contracts/SubdomainRegistrar.sol

```diff
@@ -130,6 +160,7 @@ contract SubdomainRegistrar is RegistrarInterface {
         domain.owner = owner(label);
         domain.price = 0;
         domain.referralFeePPM = 0;
+        domain.transferAddress = 0;
```

Comment:
This should not be here - or else they can back out of a commitment by unlisting the domain!